cmd/csi-adapter/cmd.go

```diff
@@ -29,10 +29,12 @@ var Version string
 
 // flags
 var (
-	identity = "cosi.storage.k8s.io"
-	nodeID   = ""
-	protocol = ""
-	listen   = ""
+	identity          = "cosi.storage.k8s.io"
+	nodeID            = ""
```

Comment:
There is no need in go to assign the "zero" value to variables, and in fact, it's a bit of an anti-pattern. This looks more like C code than typical go.  Wherever possible define your variables in their first assignment. Where this is not possible, sometimes the case in multiple func return values, then `var` is fine but don't bother with initialization unless you need to initialize the var to something other than the go zero value.

Reply:
Fixed it! I cleaned it up a bit so we just rely on the default values.